smdebug/tensorflow/utils.py

```diff
@@ -4,17 +4,16 @@
 from enum import Enum
 
 # Third Party
-from tensorflow.contrib.distribute import MirroredStrategy as ContribMirroredStrategy
+import tensorflow.compat.v1 as tf
 from tensorflow.python.distribute import values
 
 # First Party
 from smdebug.core.modes import ModeKeys
 
 try:
-    import tensorflow.compat.v1 as tf
+    from tensorflow.contrib.distribute import MirroredStrategy as ContribMirroredStrategy  # TF 1.X
```

Comment:
https://www.tensorflow.org/versions/r1.15/api_docs/python/tf/distribute/MirroredStrategy and https://www.tensorflow.org/versions/r1.15/api_docs/python/tf/contrib/distribute/MirroredStrategy are different. They have some subtle differences. Have you handled the imports fine for them?

Reply:
@jarednielsen